### ISSUE TITLE
Possible "wrong" setting of css 'max-height' for select dropdown

### ISSUE THREAD

**Issue by Grillmeisterfleisch** [w3co/jcf#30]
Hi,

I want to use the custom scrollable feature of the select dropdown. However using maxVisibleItems did not work. On inspecting the code as well as the fake elements stylings I stumbled upon a possible bug.

When the styling of the "listHolder" element is set, the code is:
`this.listHolder.css({maxHeight: this.getOverflowHeight(maxCount),overflow: 'auto'});`

However the max-height is not applied to the list holder. I've fixed this problem for me by changing `maxHeight` to `'max-height'`.

Furthermore, as the list holder element is a `<span>`-Element I would recommend to consider adding `display:'block'` by default, as `<span>`-Elements by default are inline elements which are not capable of having CSS-height stylings.

Thanks a lot.

**Comment by inferpse** [comment]
Hi @Grillmeisterfleisch !

What jQuery version are you using? I've just tried to set `maxHeight` to `<span>` element and it works just fine. You can check the current behavior of custom scrollbars by opening index.html from this repo.

Regarding the block element - JCF heavily relies on some styles which must be in the CSS (check _"theme-minimal"_ folder). All of these styles cannot be simply set inline via JS. Regarding the `listHolder` - it has `display: inline-block` style defined in CSS, so it supports height.

**Comment by Grillmeisterfleisch** [comment]
Hi @inferpse !

The jQuery Version I'm using is jQuery 2.1.4. I know it's quite old, but due to some dependencies on other parts of the project I am working on i am not able to use a newer version :-/ . I guess that's the problem why setting the max-height via 'maxHeight' did not work in my environment.

Thanks for the hint regarding the CSS dependencies of the theme minimal folder - I did not know that and guessed that all required styles to provide a basic functionality would be set inline via JS as this was the first problem I've encountered using JCF.

Thank you for your help :)
